### ISSUE TITLE
Serving Assets?

### ISSUE THREAD

**Issue by mudassir0909** [jsonresume/theme-manager#32]
What's the best way to serve assets for you theme? Right now I'm using custom font built using icomoon for theme icons, since the icon font file can't be served, I'm converting the font to base64 format & inlining it.

Can we serve any theme related assets? Perhaps a folder where the theme owners can put theme related assets & it can be served

**Comment by thomasdavis** [comment]
A public folder sounds good.

Because the themes are pushed to npm, you could also use npmcdn.com in your code to retrieve those files?

**Comment by mudassir0909** [comment]
this is awesome :)

Didn't know about npmcdn.com, we can go for it instead of serving via a public folder

**Comment by thomasdavis** [comment]
Yeah rad, I imagine we could have a resource lib later that knows how to infer the full npmcdn path by itself when in production because it would always be the same.

npmcdn.com + package.name + /public/ + resource.jpg

**Comment by mudassir0909** [comment]
yes, sounds good, we could have a handlebars helper which points to local folder during development & npmcdn during production
